src/services/tree-sitter/index.ts

```diff
@@ -5,6 +5,59 @@ import { LanguageParser, loadRequiredLanguageParsers } from "./languageParser"
 import { fileExistsAtPath } from "../../utils/fs"
 import { RooIgnoreController } from "../../core/ignore/RooIgnoreController"
 
+const extensions = [
+	"js",
+	"jsx",
+	"ts",
+	"tsx",
+	"py",
+	// Rust
+	"rs",
+	"go",
+	// C
+	"c",
+	"h",
+	// C++
+	"cpp",
+	"hpp",
+	// C#
+	"cs",
+	// Ruby
+	"rb",
+	"java",
+	"php",
+	"swift",
+].map((e) => `.${e}`)
+
+export async function parseSourceCodeDefinitionsForFile(
```

Comment:
How can we change this to include the line numbers for each definition? Feels critical

Reply:
Let me try to do it 💪🏻